mantidproject/mantid
# Fix broken clean conda builds

**Description of work.**
I broke conda clean builds in #32856 as I missed some renames from CONDA_BUILD to CONDA_ENV. My vscode had these changes but I guess they hadn't been saved. And then the CMakeCache did a fantastic job at hiding my terrible mistake 🙄 

The CONDA_BUILD variable should only exist now to modify some of the packaging calls ( as on conda build we prefer to do something different). 




**To test:**
Make a clean build in a conda env
E.g
1) rm CMakeCache.txt
2) cmake .. -preset=...
It should work


Fixes #xxxx. 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
